### PR TITLE
"typos" 🤓

### PR DESCRIPTION
**Pull request by ilyigna**
so uhm, remove mingw because vs build tools superior
and some typos i guess.

### CODE REVIEW

**Comment by lexisother** [comment]
>"vs build tools superior"
>doesn't mention the vs build tools

just leave mingw then

**Comment by Vendicated** [comment]
these are massive and also proprietary so no

**Comment by Vendicated** [comment]
Also let's be real it's not like anyone is gonna build from source on Windows

File: README.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ Download the latest Mac build from [GitHub Releases](https://github.com/Vendicat
 
 If you get a `VencordInstaller can't be opened` message, right-click `VencordInstaller.app` and click open.
 
-This error shows because the app isn't signed since I'm not paying 100 bucks for an Apple Developer license. 
+This error shows because the app isn't signed, since I'm not paying 100 bucks for an Apple Developer license.
```
Comment (x6r) [inline README.md:25]: you literally made it a typo

**Comment by KraXen72** [comment]
@Vendicated i build from source on windows with vanilla npm

**Pull request closed without merging**